### PR TITLE
Populate repository with initial project

### PR DESCRIPTION
**Pull request by LeonarddeR**
See the readme for what the proejct contains at the moment

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,23 @@
...
+
+This repository contains a .vscode directory preconfigured for the [NVDA project repository](https://github.com/nvaccess/nvda]. In the NVDA repository, it is included as a submodule.
+
+The following settings are configured in this project:
+
+* Accessibility support is enabled
+* Linting is enabled based on the Flake8 configuration bundled with the NVDA repository
+* Auto complete extra paths are added for the several external submodules
+* When saving a file, a final new line is added to it, and extra new lines are trimmed from the end of the file when applicable. This ensures a uniform style across the project
+* De default indentation is set to use tabs instead of spaces
```
Comment (feerrenrut) [inline README.md]: De -> The

**Comment by feerrenrut** [comment]
This looks really good, thanks for doing this!

**Comment by LeonarddeR** [comment]
I fixed some typos and also changed the approach to cloning inside the NVDA repo.

**Comment by LeonarddeR** [comment]
@bramd Could you have a quick look into the current project?

**Review by feerrenrut — APPROVED** [review]

**Pull request merged**